index.json

```diff
@@ -39,6 +39,31 @@
       "type": "Text Generation",
       "revision": "main"
     },
+    {
+      "name": "microsoft/phi-silica",
+      "icon": "ms",
+      "author": "Microsoft",
+      "description": "Description for Phi Silica",
+      "configPath": "finetune.yaml",
+      "availabilityLocations": [
+        {
+          "providerType": "Microsoft",
+          "modelId": "microsoft/phi-silica",
+          "revision": "main"
+        }
+      ],
+      "type": "Text Generation",
+      "revision": "main",
+      "disabledLocalFinetuning": true,
+      "supportHistory": true,
+      "versions": [
+        {
+          "model": "Phi Silica V3.6",
+          "sdk": "WinApp SDK 1.7.250127003-experimental3",
```

Comment:
Are the model version and WinAppSDK bindings correct?

Reply:
> Are the model version and WinAppSDK bindings correct?

Let's correct it when phi silica is going to public release